e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportPage.js

```diff
@@ -12,6 +12,12 @@ const utilisationReportPage = {
     generateKeyingDataButton: () => cy.get('[data-cy="generate-keying-data-button"]'),
     addAPaymentButton: () => cy.get('[data-cy="add-a-payment-button"]'),
     premiumPaymentsTable: {
+      header: {
+        totalReportedPayments: () => cy.get('th[data-cy="premium-payments-table--total-reported-payments"] button'),
+        totalPaymentsReceived: () => cy.get('th[data-cy="premium-payments-table--total-payments-received"] button'),
+        status: () => cy.get('th[data-cy="premium-payments-table--status"] button'),
+      },
+      rows: () => cy.get('tr[data-cy^="premium-payments-table-row--feeRecordId-"]'),
```

Comment:
Looks like a typo here? `feeRecordId-`

Reply:
This is to get all rows with that prefix (which is then followed by the specific fee record ID) - added a comment just above this line